src/main/resources/.properties

```diff
@@ -0,0 +1,2 @@
+version=${project.version}
```

Comment:
Is the project version used for anything?  Also, is there a reason this is tracked and the git properties file is not?  I'm a little hesitant to have yet another "config" file, but I'm guessing this is the only one to pull info from the `pom.xml`?

Reply:
I'll remove the version field.  The git.properties file is autogenerated and will change with every commit, so that shouldn't be checked it.